src/thunderbots/software/ai/navigator/path_planner/astar.h

```diff
@@ -0,0 +1,149 @@
+#pragma once
+
+// I apologize if your CLion slows to a crawl
```

Comment:
:laughing:

Reply:
I've had CLion completely lock up at least 5 times so far

Reply:
Really? I didn't use it for very long with this, so that's likely why I didn't experience a lockup. If you have a fairly consistent repro, would you mind submitting a ticket to CLion? I've filed a few in the past and they've been resolved.

https://youtrack.jetbrains.com/issues/CPP

Reply:
changed